lib/seattleflu/id3c/cli/command/clinical.py

```diff
@@ -336,12 +336,12 @@ def add_kp_manifest_data(df: pd.DataFrame, manifest_filenames: tuple) -> pd.Data
     """
     manifest_data = pd.DataFrame()
     dtypes = {
-        'sample_id': 'string',
-        'kp_id': 'string'
+        'sample_id': str,
+        'kp_id': str
```

Comment:
Ok, I see. I was following examples I saw in Pandas [documentation](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_excel.html) that use `str` instead of `"string"`

It looks like this is a [relevant change](https://pandas.pydata.org/pandas-docs/stable/whatsnew/v1.1.0.html#all-dtypes-can-now-be-converted-to-stringdtype) in 1.1. But [here](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.StringDtype.html#pandas.StringDtype) it also says StringDType is still experimental.

I can change it back if that's the preference but I wanted to better understand the difference. At this point it looks like either will work.

Reply:
Ah, right @tsibley actually had to do [some extra work](https://github.com/seattleflu/id3c/pull/116) to make `"string"` work as intended. Maybe we should use our ID3C `read_excel` function here to ensure we avoid the same pitfalls.

Reply:
I replaced the `pandas.read_excel` function with the version in ID3C. The `pandas.read_excel` function is also used in the `parse-uw` but I did not change it there since this branch is specifically for Kaiser data.